projects/1.18/assets/monsters-girls-mushroom-update/monsters_and_girls/lang/zh_cn.json

```
@@ -0,0 +1,225 @@
+{
+  "biome.monsters_and_girls.spooky_forest": "幽灵森林",
+  "block.monsters_and_girls.blockof_ender_puffbal": "末影尘菌块",
+  "block.monsters_and_girls.end_puffball": "末影尘菌",
+  "block.monsters_and_girls.ender_moss": "末影苔藓",
+  "block.monsters_and_girls.ender_mushroom_stem": "末影蘑菇柄",
+  "block.monsters_and_girls.ender_puffball_button": "末影尘菌按钮",
+  "block.monsters_and_girls.ender_puffball_doors": "末影尘菌门",
+  "block.monsters_and_girls.ender_puffball_fence": "末影尘菌栅栏",
+  "block.monsters_and_girls.ender_puffball_gate": "末影尘菌栅栏门",
+  "block.monsters_and_girls.ender_puffball_planks": "末影尘菌木板",
+  "block.monsters_and_girls.ender_puffball_plate": "末影尘菌压力板",
+  "block.monsters_and_girls.ender_puffball_slab": "末影尘菌台阶",
+  "block.monsters_and_girls.ender_puffball_stairs": "末影尘菌楼梯",
+  "block.monsters_and_girls.ender_puffball_trapdoor": "末影尘菌活板门",
+  "block.monsters_and_girls.ender_shroomlight": "末影菌光体",
+  "block.monsters_and_girls.glow_berry_bush": "发光浆果丛",
+  "block.monsters_and_girls.glow_berry_bush_unlit": "发光浆果丛",
+  "block.monsters_and_girls.huge_brown_mushroom": "巨大的棕色蘑菇",
+  "block.monsters_and_girls.huge_crimson_fungus": "巨大的绯红菌",
+  "block.monsters_and_girls.huge_ender_puffball": "巨大的末影尘菌",
+  "block.monsters_and_girls.huge_green": "巨大的罕见诡异菌",
+  "block.monsters_and_girls.huge_infernal_mushroom": "巨大的地狱蘑菇",
+  "block.monsters_and_girls.huge_inky_cap": "巨大的墨汁鬼伞",
+  "block.monsters_and_girls.huge_magma_fungus": "巨大的熔融真菌",
```

```suggestion
  "block.monsters_and_girls.huge_magma_fungus": "巨大的熔融菌",
```
和绯红菌诡异菌翻译格式统一
此处更改则下方也一起更改